c-api/memoryview.po

```diff
@@ -32,19 +36,28 @@ msgid ""
 "will be read/write, otherwise it may be either read-only or read/write at "
 "the discretion of the exporter."
 msgstr ""
+"Crée un objet memoryview à partir d'un objet qui fournit l'interface tampon. "
+"si *obj* permet des exports d'interface tampon, l'objet memoryview peut être "
```

Comment:
`If *obj* supports writable buffer exports` n'est pas évident à traduire. Dans ta traduction il manque la notion de `writable`.

Dans tous les cas l'objet permet l'export, sans quoi on ne peut pas créer un memoryview. L'export c'est le fait d'exposer un buffer via une API. Cette API fournit un *buffer*, ce *buffer exporté* peut soit etre en lecture seule, en ce cas la memoryview sera en lecture seule, soit être en lecture/écriture en ce cas la memoryview sera en lecture/écriture.

Il faut, je pense, lire https://docs.python.org/3/c-api/buffer.html avant, qui explique ce que signifie "exporter un buffer" :

> a type can export a “buffer interface” which allows objects of that type to expose information about their underlying buffer.

Reply:
La doc dit `Any object that can export a series of bytes through the buffer interface can be written to a file`.

On pourrait faire cela : `Si *obj* permet les exports d'interfaces tampon en lecture comme en écriture notamment lors d'un write()`. Qu'en penses-tu ?